### PR TITLE
WIP: Render first 3 homeworks for Ostrava from YAML

### PR DESCRIPTION
**Pull request by encukou**
Takes the task definitions in YAML/Markdown from [Courseware](https://github.com/messa/pyladies-courseware) and renders them in HTML.

Further homework can be added similarly – 3 files in `lessons/homework/*` and replacing the entry in `runs/*/info.yml`.

---
WIP: Requires naucse_render 1.1+ with merged pull request:
    https://github.com/pyvec/naucse_render/pull/6
(Should be merged & released before merging this.)

Run "pipenv install" to update.

### CODE REVIEW

**Comment by frenzymadness** [comment]
Could you please add here all the rest?

**Comment by encukou** [comment]
> Could you please add here all the rest?

It is on my longer-term TODO, but I'd like this  merged now, so it's on the website (and so others can possibly help adding the new task sets).

**Comment by frenzymadness** [comment]
Okay. I told our girls that the form of home projects might change slightly in the next weeks so from my point of view is okay to deliver it step by step.

@Glutexo might find some time to help with this when traveling across boring USA :smile: 

I'll do a final review of content in a minute.

File: lessons/homework/ostrava3/tasks.yml [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+    _Od ornamentů ke spirále se dostaneš stejným způsobem jako od `n`-úhelníků ke kolečku._
+
+- markdown: |
+    Napiš obrázek! Třeba les ze stromů, město z domů, nebe z hvězd,
+    dav ze smajlíků, hradbu z věží, zikkurat ze schodů, ...
+
+    Můžeš použít i nějakou z funkcí modulu `random`.
+
+    Podle toho, jak jste se na sraze domluvili, pošli řešení e-mailem
```
Comment (frenzymadness) [inline lessons/homework/ostrava3/tasks.yml]: I'd remove this note as you did in the second handout. In Ostrava, girls are sending all home projects to the dropbox.

**Comment by frenzymadness** [comment]
Just one comment. Otherwise, it looks good to me. Merge at will.

**Pull request merged**